jkube-kit/generator/api/src/test/java/org/eclipse/jkube/generator/api/support/BaseGeneratorTest.java

```diff
@@ -44,16 +44,16 @@
 import static org.junit.Assert.assertThrows;
 import static org.junit.Assert.assertTrue;
```

Comment:
Please remove all these JUnit4 assertions and use assertJ instead. `assertThat(..).isEqualTo()`, `assertThat(..).isFalse()` etc.